chromium_src/chrome/browser/ui/webui/help/version_updater_mac.h

```diff
@@ -59,7 +61,9 @@ class VersionUpdaterMac : public VersionUpdater {
   // The observer that will receive Keystone status updates.
   KeystoneObserver* __strong keystone_observer_;
 
-  base::WeakPtrFactory<VersionUpdaterMac> weak_factory_{this};
+  bool (*type_marker_)(VersionUpdater*) = &SparkleVersionUpdater::IsSparkle;
```

Comment:
I assume that you use this marke as we can't use `dynamic_cast` for testing?

Reply:
Exactly. I've added a comment that describes this in aaa11a383fbc13d9db257a31b35631822adbd9be.